internal/ingress/controller/config/config.go

```diff
@@ -528,6 +534,10 @@ func NewDefault() Configuration {
 	defIPCIDR = append(defIPCIDR, "0.0.0.0/0")
 	defNginxStatusIpv4Whitelist = append(defNginxStatusIpv4Whitelist, "127.0.0.1")
 	defNginxStatusIpv6Whitelist = append(defNginxStatusIpv6Whitelist, "::1")
+	defProxyDeadlineDuration, err := time.ParseDuration("5s")
```

Comment:
You could assign the variable to a `time.Duration` directly and remove any possibility of failing at run time due to an incorrect input. eg. `time.Duration(5000000000)` or (more readable) `time.Duration(5) * time.Second`.

Reply:
Good catch. This should have been my initial approach